examples/sasrec_metrics_comp.ipynb

```diff
@@ -987,15 +1040,20 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 24,
    "metadata": {},
    "outputs": [
     {
      "name": "stderr",
      "output_type": "stream",
      "text": [
       "/data/home/maspirina1/Tasks/RecTools/rectools/dataset/features.py:424: UserWarning: Converting sparse features to dense array may cause MemoryError\n",
-      "  warnings.warn(\"Converting sparse features to dense array may cause MemoryError\")\n"
+      "  warnings.warn(\"Converting sparse features to dense array may cause MemoryError\")\n",
+      "/data/home/maspirina1/Tasks/RecTools/rectools/models/base.py:406: UserWarning: \n",
+      "                Model `<class 'rectools.models.implicit_als.ImplicitALSWrapperModel'>` doesn't support recommendations for cold users,\n",
+      "                but some of given users are cold: they are not in the `dataset.user_id_map`\n",
```

Comment:
Please  check which users are considered cold by ALS. This is an unexpected behaviour and we need to find a reason.

Reply:
Test users passed to ALS were actually cold. In new sasrec_metrics_comp.ipynb I pass all users to SASRec and no cold users to ALS.